Description: Upstream changes introduced in version 3.5-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 units-filter (3.5-2) unstable; urgency=low
 .
   * fixed typos in the manpage, thanks to A. Costa's patch.
     Closes: #649228
 .
 The person named in the Author field signed this changelog entry.
Author: Georges Khaznadar <georgesk@ofset.org>
Bug-Debian: http://bugs.debian.org/649228

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- units-filter-3.5.orig/manpage.xml
+++ units-filter-3.5/manpage.xml
@@ -122,16 +122,16 @@ standard SI unit, followed by the physic
       <varlistentry>
         <term><option>-s </option></term>
         <listitem>
-          <para>Like Significative.</para>
+          <para>Like Significant.</para>
 	  <para>
-	    Takes in account the number of significative digits. For
-example 1.0 m contains 2 significative digits, while 0.00100 contains
-3 significative digits.  It is possible to enforce the number of
-significative digits by using a special syntax : if units-filter
+	    Takes in account the number of significant digits. For
+example 1.0 m contains 2 significant digits, while 0.00100 contains
+3 significant digits.  It is possible to enforce the number of
+significant digits by using a special syntax : if units-filter
 parses the input "1.0m#6", it interprets it as a value with exactly 6
-significative digits, like "1.00000 m". The number following the #
-sign is the forced number of significative digits. The number of
-significative digits appears just before the last zero in the output
+significant digits, like "1.00000 m". The number following the #
+sign is the forced number of significant digits. The number of
+significant digits appears just before the last zero in the output
 of the command (this zero is a placeholder for future extensions).
 	  </para>
         </listitem>
@@ -145,7 +145,7 @@ physical unit in the International Syste
 simplification with usual units. For example, a newton will be represented by
 the unit N in place of m.kg.s^-2. The value is expressed as a floating number
 with one digit before the decimal point, and as many digits in the mantissa as
-necessary to fit the desired number of significative digits (see an example
+necessary to fit the desired number of significant digits (see an example
 below). It is possible to enforce the output unit : just add a colon and the 
 desired unit at the end of the input. If this unit is homogeneous with the
 former one, it will be used to format the output.
@@ -167,19 +167,19 @@ in LaTeX language.
   <refsect1 id="examples">
     <!-- Or use this section to tell about upstream BTS. -->
     <title>EXAMPLES</title>
-    <para>Establishing the SI value and units exponents of a quantity in the mksa system:</para>
+    <para>Establish the SI value and unit exponent of a quantity in the mksa system:</para>
 <para><command>~$ echo 1.5e3 nN.m.s^-1 | units-filter</command></para>
 <para><command>1.5e-6   2   1  -3   0   0   0   0</command></para>
 <para>which means : 1.5e-6 (SI unit) m^2.kg.s^-3</para>
 <para></para>
-<para>Comparing to different physical quantities:</para>
+<para>Compare different physical quantities:</para>
 <para><command>~$ e1=$(echo "1.2e-3 V" | units-filter)</command></para>
 <para><command>~$ e2=$(echo "1200e3  nWb/s"| units-filter)</command></para>
 <para><command>~$ if [ "$e1" = "$e2" ]; then echo ok; else echo ko; fi</command></para>
 <para><command>ok</command></para>
-<para> ... which emphasizes that webers by unit time are the same than volts.</para>
+<para> ... which emphasizes that webers by unit time are the same as volts.</para>
 <para></para>
-<para>Playing with the number of significative digits:</para>
+<para>Playing with the number of significant digits:</para>
 <para><command>~$ echo "0.00100m" | src/units-filter -s</command></para>
 <para><command>0.001   1   0   0   0   0   0   0    3    0</command></para>
 <para><command>~$ echo "0.00100m #2" | src/units-filter -s</command></para>
@@ -203,7 +203,7 @@ in LaTeX language.
   </refsect1>
   <refsect1>
     <title>KNOWN BUGS</title>
-<para>Few units out of the mksa system are sucessfuly parsed.</para>
+<para>Few units out of the mksa system are successfully parsed.</para>
   </refsect1>
 </refentry>
 
